cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -65,4 +53,70 @@ describe('Moderation Tools for Posts', () => {
       });
     });
   });
+
+  it('should show Feature Post button on an unfeatured post for an admin user', () => {
```

Comment:
@aitchiss I tried to place the admin-role-related specs on lines `57` and `73` within a `context` block (similar to the trusted-user-related specs on line `89`), but the tags spec kept failing 🤷🏾‍♀️ 

I welcome any ideas to make this arrangement work (or we can just leave the specs as-is 😅 )

Reply:
Hmm it shouldn't affect other specs but I can't tell off the top of my head what was going wrong 😅 

I don't mind whether it stays as-is or whether we wrap in a context... feel free to push up the version with context and I'll take a look at what's going on though 🙂